pkg/kubernetes/tool.html

```diff
@@ -99,6 +99,47 @@
         return null;
     }
 
+    $("#create-minion-dialog").on("show.bs.modal", function(event) {
```

Comment:
It seems like this accidentally snuck into this commit. It doesn't have to do with deleting.